### PR TITLE
[4.0] alphabetical sorting of the webservices strings in com_config.ini

### PR DESCRIPTION
**Pull request by tecpromotion**
### Summary of Changes

alphabetical sorting of the webservices strings in com_config.ini

### CODE REVIEW

File: administrator/language/en-GB/com_config.ini [hunk 1]
```diff
@@ -177,6 +177,10 @@ COM_CONFIG_FIELD_VALUE_SMTP="SMTP"
 COM_CONFIG_FIELD_VALUE_SSL="SSL/TLS"
 COM_CONFIG_FIELD_VALUE_SYSTEM_DEFAULT="System Default"
 COM_CONFIG_FIELD_VALUE_TLS="STARTTLS"
+COM_CONFIG_FIELD_WEBSERVICES_CORSOFFF_LABEL="Enable CORS"
+COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_ORIGIN_DESC="Specifies the origin allowed to access webservices on this site,sent back in response to a preflight request. Default: * (=all)."
+COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_HEADERS_DESC="Specifies the header(s) sent back in response to a preflight request. Default: Content-Type,X-Joomla-Token"
```
Comment (Quy) [inline administrator/language/en-GB/com_config.ini:182]:
```suggestion
COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_HEADERS_DESC="Specifies the header(s) sent back in response to a preflight request. Default: Content-Type,X-Joomla-Token"
COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_ORIGIN_DESC="Specifies the origin allowed to access webservices on this site,sent back in response to a preflight request. Default: * (=all)."
```

File: administrator/language/en-GB/com_config.ini [hunk 2]
```diff
@@ -177,6 +177,10 @@ COM_CONFIG_FIELD_VALUE_SMTP="SMTP"
...
+COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_METHODS_DESC="Specifies the webservice method(s) allowed to access on this site, sent back in response to a preflight request. Default: all methods available for the requested route."
```
Comment (Quy) [inline administrator/language/en-GB/com_config.ini]:
```suggestion
COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_HEADERS_DESC="Specifies the header(s) sent back in response to a preflight request. Default: Content-Type,X-Joomla-Token"
COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_METHODS_DESC="Specifies the webservice method(s) allowed to access on this site, sent back in response to a preflight request. Default: all methods available for the requested route."
COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_ORIGIN_DESC="Specifies the origin allowed to access webservices on this site,sent back in response to a preflight request. Default: * (=all)."
```

Comment (tecpromotion) [reply administrator/language/en-GB/com_config.ini]: I just noticed that too. Thank you @Quy

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 290fca0063caf37b14c95b9a9e80bf81efbb7248<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32211">issues.joomla.org/tracker/joomla-cms/32211</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 290fca0063caf37b14c95b9a9e80bf81efbb7248<br /><br />By review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32211">issues.joomla.org/tracker/joomla-cms/32211</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32211">issues.joomla.org/tracker/joomla-cms/32211</a>.</sub>

**Comment by zero-24** [comment]
Merging thanks.

**Pull request merged**